### ISSUE TITLE
Mouse one-way border crossing [bug]

### ISSUE THREAD

**Issue by ammone95** [mgth/LittleBigMouse#315]
Hello,
I'm having trouble with moving the cursor from a display to another. See attachment for reference and settings.

![littlebigmouse_bug_1](https://github.com/mgth/LittleBigMouse/assets/50293346/5b1bdb1c-7fc9-4200-baa6-7551c11c8635)

i have a laptop (3) with two monitors, one above horizontal (1) and a vertical one on the right (2).
it seems that i can't go from horizontal monitor to laptop but reverse can.
Basically i can go back and forward from 3 to 2 and 2 to 3, same from 1 to 2 and 2 to 1 but with 3 and 1 i can only go in one direction.

I could use the corner crossing algorithm but it's a bit buggy, whenever detects a slight diagonal movement the mouse skips very far on the monitor.

App version 5.0.4.1

**Comment by ammone95** [comment]
Issue fixed in 5.0.5.0
